docs/learn/learn-staking.md

```diff
@@ -9,189 +9,357 @@ slug: ../learn-staking
+security of the network depends on the amount of capital locked on the chain: the more the capital
+locked, the lower the chance of an attack on the network, as the attacker
+needs to incur a heavy loss to orchestrate a successful attack (more on this later on). The process of
+locking tokens on the chain is called `staking`.
+
+Similar to the miners in PoW networks, PoS networks have `validators`, but they do not have to compete with each other to solve mathematical puzzles.
+They are instead pre-selected to produce the blocks based on the stake backing them.
+Token holders can lock funds on the chain and for doing so, they are getting `staking rewards`. There is
+thus an economic incentive for token holders to become active participants who contribute to the
+security and economic stability of the network. PoS networks in general are therefore more inclusive
+than PoW networks, as participants do not need to have either technical knowledge about
+blockchain technology or experience in running mining equipment. 
+
+PoS ensures that everybody participating in the staking process has
+"skin in the game" and thus can be held accountable. In case of misbehavior, participants in the
+staking process can be punished or `slashed`, and depending on the gravity of the situation, their
+stake can be partly or fully confiscated by the network. It is not in a staker's economic interest to orchestrate an attack and risk losing tokens. Any rational actor staking on the network would want to get rewarded, and the PoS network rewards good behavior and punishes bad behavior.
+
+## Nominated Proof-of-Stake Overview
+
+{{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }} implements [Nominated Proof-of-Stake (NPoS)](learn-consensus.md/#nominated-proof-of-stake),
+a relatively novel and sophisticated mechanism to select the validators who are allowed to participate in 
+its [consensus](learn-consensus.md) protocol. The NPoS encourages {{ polkadot: DOT :polkadot }}{{ kusama: KSM :kusama }} 
+holders to participate as nominators. NPoS encourages {{ polkadot: DOT :polkadot }}{{ kusama: KSM :kusama }}
+holders to participate as `nominators`. 
+
 Nominators may back up to
 {{ polkadot: <RPC network="polkadot" path="consts.staking.maxNominations" defaultValue={16}/> :polkadot }}
 {{ kusama: <RPC network="kusama" path="consts.staking.maxNominations" defaultValue={24}/> :kusama }}
-validators as trusted validator candidates. Both validators and nominators lock their tokens as
-collateral and receive staking rewards.
+validators as trusted validator candidates, and the network will automatically distribute the stake among validators in
```

Comment:
autoamtically, such that the economic security is maximized. 

And ideally "economic security" should be explained.